exonum/src/api/private/unsafe_api.rs

```diff
@@ -0,0 +1,64 @@
+// Copyright 2018 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Unsafe Exonum api that provides functionality for emergency cases.
+
+use router::Router;
+use iron::prelude::*;
+
+use std::process;
+
+use blockchain::{Blockchain, Schema};
+use api::{Api, ApiError};
+
+/// Private unsafe API.
+#[derive(Clone, Debug)]
+pub struct UnsafeApi {
+    blockchain: Blockchain,
+}
+
+impl UnsafeApi {
+    /// Creates a new `private::UnsafeApi` instance.
+    pub fn new(blockchain: Blockchain) -> UnsafeApi {
```

Comment:
Me and @DarkEld3r prefer using `Self` where possible :upside_down_face: 
```rust
pub fn new(...) -> Self {
    Self { ... }
}
```

Reply:
Good remark. But I guess then it'll be a good idea to change it in private::SystemApi and public::SystemApi too.

Reply:
Let's create separate task for this (for external contributors, I suppose) :wink: